xarray/core/accessor_dt.py

```diff
@@ -323,3 +240,140 @@ def strftime(self, date_format):
         return obj_type(
             result, name="strftime", coords=self._obj.coords, dims=self._obj.dims
         )
+
+
+class DatetimeProperties(Properties):
```

Comment:
Let's keep this `DatetimeAccessor` and the enxt one `TimedeltaAccessor`. It's in line with our documentation terminology that way. `TimedeltaAccessor` should be added to `api.rst`

Reply:
Sounds good. And I assume we still want both `DatetimeAccessor` and `TimedeltaAccessor` to be accessed via `.dt`, right?

Reply:
Yes. that's consistent with Pandas right?

Reply:
Yes, it's.

Reply:
OK let's do that.